### PR TITLE
Adding vscode support

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: vscode/language-configuration.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "comments": {
+    // symbol used for single line comment. Remove this entry if your language does not support line comments
+    "lineComment": "#"
+  },
+  // symbols used as brackets
+  "brackets": [
+    ["{", "}"],
```
Comment (cjllanwarne) [inline vscode/language-configuration.json]: do we need the `${` and `}` pair too?

Comment (ghost) [reply vscode/language-configuration.json]: Added below

**Comment by cjllanwarne** [comment]
Do we know what's happening with the travis failures? IMO constant-red is worse than no tests at all

**Comment by ghost** [comment]
It's attempting to publish to the VSCode marketplace with a garbage personal access token and failing. No tests, just automated the publish step.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-Installation:
+ ### Installation:
+ #### Sublime Text:
   * Install [Package Control](https://packagecontrol.io/) package manager for Sublime via [these instructions](https://packagecontrol.io/installation).
   * Use <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on OSX or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on Linux to open Command Palette, then select `Package Control: Install Package`.
   * Search for "WDL Syntax" and hit <kbd>Enter</kbd>. 
-  
+ 
+ #### Visual Studio Code:
+ * Go to Extensions and search for WDL Syntax Highlighter and install!
```
Comment (cjllanwarne) [inline README.md]: In a world where we don't have a MS account, is there any way to manually install a pre-packaged release?

File: vscode/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "wdl",
+  "displayName": "WDL Syntax Highlighter",
+  "description": "Official Syntax Highlighter for the Workflow Definition Language",
+  "version": "0.0.1",
```
Comment (cjllanwarne) [inline vscode/package.json]: ToL (thinking out loud, ie feel free to decide you disagree): I'm not a huge fan of "never quite 1.0" versions. If we think this is good enough to install we can probably just call it 1.0 and increment from there

**Comment by cjllanwarne** [comment]
Not sure what to do with this PR since (as I understand things) it doesn't add anything until we have a MS account, and at that point things will still have to change before it's useful.

I'll leave it up to you whether to add this now in preparation, or leave the PR open until we get everything ready to go in one commit.

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: I will look into finding a simple way and update this doc.

File: vscode/package.json (hunk 3 above)
Comment (ghost) [reply vscode/package.json]: I agree with you. Changing to 1.0.0.

**Comment by ghost** [comment]
I'll leave the PR open as BITS is taking a while to respond and I need to edit the README with better instructions for installing via our github repo as opposed to the vscode marketplace.

File: README.md [hunk 4]
```diff
@@ -1,9 +1,13 @@
-## [WDL](https://software.broadinstitute.org/wdl/) syntax for Sublime Text editor
+## [WDL](https://software.broadinstitute.org/wdl/) syntax for Sublime Text Editor and VS Code
 
-This repo provides support for the [Workflow Development Language (WDL)](https://software.broadinstitute.org/wdl/) in the [Sublime Text Editor](https://www.sublimetext.com/).
+This repo provides support for the [Workflow Description Language (WDL)](https://software.broadinstitute.org/wdl/) in the [Sublime Text Editor](https://www.sublimetext.com/) 
+and [Visual Studio Code](https://code.visualstudio.com/).
```
Comment (cjllanwarne) [inline README.md]: And `github/linguist`, as it happens...

File: vscode/package.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        ],
+        "grammars": [
+            {
+                "language": "wdl",
+                "scopeName": "source.wdl",
+                "path": "./WDL.tmLanguage"
+            }
+        ]
+    }
+}
```
Comment (cjllanwarne) [inline vscode/package.json]: Add newline

**Review by cjllanwarne — APPROVED** [review]
A couple of minor comments, but otherwise ready for squash & merge

**Pull request merged**